manifests/windows/install.pp

```diff
@@ -23,6 +23,11 @@
     }
   }
 
+  $_cmd_location = $::rubyplatform ? {
+    /i386/  => "C:\\Windows\\system32\\cmd.exe",
+    default => "${::system32}\\cmd.exe"
+  }
```

Comment:
Either way, this should always come out to `C:\Windows\system32\cmd.exe` - this can be simplified by removing it.